python/utils/CMakeLists.txt

```diff
@@ -9,6 +9,9 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
 SET(CMAKE_INSTALL_RPATH "$ORIGIN:$ORIGIN/..")
 
 add_library(cudaq-py-utils SHARED LinkedLibraryHolder.cpp)
-target_link_libraries(cudaq-py-utils PRIVATE nvqir cudaq fmt::fmt-header-only CUDAQTargetConfigUtil)
-target_include_directories(cudaq-py-utils PUBLIC $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/python/utils>)
+target_link_libraries(cudaq-py-utils PRIVATE nvqir cudaq fmt::fmt-header-only CUDAQTargetConfigUtil cudaq-common)
```

Comment:
nit: is this longer than 80 columns?

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/pull/3292/commits/926a15b80a0150cc45270c4fa7f5e75d0ab9d821